elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -199,18 +197,97 @@ default void preCommit() {
         return val;
     }
 
-    default <T> Object getRelation(
+    /**
+     * @param relationTx - The datastore that governs objects of the relationhip's type.
+     * @param entity - The object which owns the relationship.
+     * @param relationName - name of the relationship.
+     * @param filterExpression - filtering which can be pushed down to the data store.
+     *                         It is optional for the data store to attempt evaluation.
+     * @param sorting - sorting which can be pushed down to the data store.
+     * @param pagination - pagination which can be pushed down to the data store.
+     * @param scope - contains request level metadata.
+     */
+    default Object getRelation(
+            DataStoreTransaction relationTx,
             Object entity,
-            RelationshipType relationshipType,
             String relationName,
-            Class<T> relationClass,
-            EntityDictionary dictionary,
             Optional<FilterExpression> filterExpression,
-            Sorting sorting,
-            Pagination pagination
-    ) {
-        return PersistentResource.getValue(entity, relationName, dictionary);
-    }
+            Optional<Sorting> sorting,
```

Comment:
The default method for this should call relationTx.load with a FilterExpression on the parent entity ID.